consensus/ibft/sign.go

```diff
@@ -127,8 +127,9 @@ func calculateHeaderHash(h *types.Header) ([]byte, error) {
 		return nil, err
 	}
 
-	// This will effectively remove the Seal and Committed Seal fields, while keeping proposer vanity and validator set
-	// 		because extra.Validators is what we got from `h` in the first place.
+	// This will effectively remove the ShouldSeal and Committed ShouldSeal fields,
```

Comment:
Typo in search-replace probably

Reply:
Fixed 👍 

[f1405b1](https://github.com/0xPolygon/polygon-edge/pull/410/commits/f1405b160f94a379db4fd3f0bbb4c6419d674d2e)